http-clients/src/main/java/com/palantir/remoting/http/FeignClientFactory.java

```diff
@@ -134,6 +139,41 @@ public static FeignClientFactory of(
     }
 
     /**
+     * Constructs a dynamic proxy for the specified type using the supplied {@link ServiceConfiguration}. If the SSL
+     * configuration is not present in the {@link ServiceConfiguration}, the fallback SSL configuration will be used.
+     */
+    public <T> T createProxies(
+            ServiceConfiguration serviceConfig,
+            Optional<SslConfiguration> fallbackSslConfiguration,
+            Class<T> serviceClass) {
+
+        Optional<SSLSocketFactory> socketFactory = Optional.absent();
+        Optional<SslConfiguration> sslConfig = serviceConfig.security().or(fallbackSslConfiguration);
+
+        if (sslConfig.isPresent()) {
+            socketFactory = Optional.of(SslSocketFactories.createSslSocketFactory(sslConfig.get()));
+        }
+
+        Set<String> uris = Sets.newHashSet(serviceConfig.uris());
+        return FeignClients.standard().createProxy(socketFactory, uris, serviceClass);
```

Comment:
should use methods from this FeignClientFactory instance rather than FeignClients.standard()